src/icons/fileIcons.ts

```diff
@@ -471,6 +471,7 @@ export const fileIcons: FileIcons = {
         { name: 'haml', fileExtensions: ['haml'] },
         { name: 'yang', fileExtensions: ['yang'] },
         { name: 'mjml', fileExtensions: ['mjml'] },
+        { name: 'now', fileNames: ['now.json'], light: true },
```

Comment:
Could you please also add the file name `.nowignore` as it was done in the other pull request:

https://github.com/PKief/vscode-material-icon-theme/blob/7758cff3f2480cd8ca22b9019857c779b9f89758/src/icons/fileIcons.ts#L793

Reply:
done